addon/components/bs-form/element.js

```diff
@@ -271,6 +271,24 @@ export default class FormElement extends FormGroup {
    * @property value
    * @public
    */
+  get value() {
+    if (this.property && this.model) {
+      return get(this.model, this.property);
+    }
+    return this._value;
+  }
+
+  set value(value) {
+    this._value = value;
+  }
```

Comment:
Wondering if Ember Bootstrap sets `value` itself or if the setter is only needed to support `@value` argument until the component is refactored to `@glimmer/component`. In the latter case I would move the assertion that not `@value` and `@property` could not be set at the same time to this setter. It would make it a little bit easier to follow the code in my opinion.

Reply:
It's the latter, and yes, I agree.